### PR TITLE
EVG-2793 Commit display shows commit year as 0000

### PR DESCRIPTION
**Pull request by richardsamuels**

### CODE REVIEW

File: thirdparty/github_models.go [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
+
+	b := []byte{0x22}
+	b = append(b, []byte(timeStr)...)
+	b = append(b, 0x22)
+
+	return b, nil
+}
+
+func (t *GithubTime) Time() time.Time {
+	return time.Time(*t)
```
Comment (tychoish) [inline thirdparty/github_models.go]: can you not just return *t here?

File: thirdparty/github_models.go [hunk 2]
```diff
@@ -1,9 +1,58 @@
...
+
+	return nil
+}
+
+func (t *GithubTime) MarshalJSON() ([]byte, error) {
+	timeStr := t.Time().Format(githubTimeFormat)
+
+	b := []byte{0x22}
+	b = append(b, []byte(timeStr)...)
+	b = append(b, 0x22)
```
Comment (tychoish) [inline thirdparty/github_models.go]: we've been talking offline about solutions, I think my current favorite is: 

    []byte(fmt.Sprintf(`"%s"`, timeStr))

but something that avoids raw hardcoded bytes

File: thirdparty/github_models.go (hunk 1 above)
Comment (richardsamuels) [reply thirdparty/github_models.go]: no, the cast is required

File: thirdparty/github_models.go (hunk 2 above)
Comment (richardsamuels) [reply thirdparty/github_models.go]: done

**Pull request merged**